lib/tasks/fixups.rake

```diff
@@ -3,6 +3,28 @@ require 'rake'
 namespace :app do
   namespace :convert do
 
+    desc 'Fix user fields which are not compliant with ActiveRecord validation checks.'
+    task :fix_invalid_user_fields => :environment do
+        regex = /(?=\A[^[:cntrl:]\\<>\/&]*\z)(.*[\p{L}\d].*)/u
+        count = 0
+        User.find_each do |u|
+            if !(u.first_name =~ regex && u.last_name =~ regex)
+                puts "id: #{u.id} login: #{u.login} "   +
+                        "first name: #{u.first_name} "  +
+                        "last name: #{u.last_name} "    +
+                        "email #{u.email}"
```

Comment:
In order to make the results more easily converted into an email blast. It would be better to output them in CSV style. So output a header of : ID, Login, First Name, Last Name, Email and then follow that with a row for each invalid user.